haystack/nodes/answer_generator/openai.py

```diff
@@ -56,12 +56,13 @@ def __init__(
         max_tokens: int = 13,
         top_k: int = 5,
         temperature: float = 0.2,
-        presence_penalty: float = -2.0,
-        frequency_penalty: float = -2.0,
+        presence_penalty: float = 0.1,
+        frequency_penalty: float = 0.1,
         examples_context: Optional[str] = None,
-        examples: Optional[List] = None,
-        stop_words: Optional[List] = None,
+        examples: Optional[List[List[str]]] = None,
+        stop_words: Optional[List[str]] = None,
         progress_bar: bool = True,
+        instruction_prompt: Optional[str] = None,
```

Comment:
Would feel more consistent with the PromptNode design, if we introduced here `prompt_template=Optional[PromptTemplate]`

Reply:
Hey, @tholor I've updated the node to use `PromptTemplate`. The final prompt is the same as before, but the user has more control over the structure of the prompt and what can be specified.